### PR TITLE
Adding new footer icons (StarCitizenTools style) to lhmnwiki and sister projects

### PR DESCRIPTION
**Pull request by songnguxyz**
Adding new footer icons (StarCitizenTools style)

### CODE REVIEW

**Comment by clarfonthey** [comment]
Is there not a setting to modify these under `ManageWiki`? It makes sense if people are using different images to match a certain style; either we should have a policy of these images remaining the same, or letting people modify them on the wikis themselves.

**Comment by songnguxyz** [comment]
> Is there not a setting to modify these under `ManageWiki`? It makes sense if people are using different images to match a certain style; either we should have a policy of these images remaining the same or letting people modify them on the wikis themselves.

I'd love to if that would come true, but the support for it hasn't been implemented yet.

**Comment by RhinosF1** [comment]
> > Is there not a setting to modify these under `ManageWiki`? It makes sense if people are using different images to match a certain style; either we should have a policy of these images remaining the same or letting people modify them on the wikis themselves.
> 
> I'd love to if that would come true, but the support for it hasn't been implemented yet.

Is there any reason you couldn't make it come true?

**Comment by clarfonthey** [comment]
I should have also clarified, asking why this isn't a setting wasn't necessarily directed at you (songnguxyz), but mostly the folks viewing it in general. I think it would be lovely to implement if it's desired.

**Comment by songnguxyz** [comment]
Accidentally closed.

**Comment by songnguxyz** [comment]
Oh come on.

File: LocalSettings.php [hunk 1]
```diff
@@ -1588,11 +1588,31 @@
 					'src' => 'https://static.miraheze.org/lhmnwiki/5/58/Footer.SN.xyz.svg',
 					'url' => 'https://songngu.xyz',
 					'alt' => 'Dự án được bảo quản bởi SongNgư.xyz',
+					"height" => "32",
+				        "width" => "200"
 				],
 				'miraheze' => [
-					'src' => 'https://static.miraheze.org/commonswiki/f/ff/Powered_by_Miraheze.svg',
+					'src' => 'https://static.miraheze.org/lhmnwiki/1/1c/Miraheze.svg',
```
Comment (Reception123) [inline LocalSettings.php:1595]: is there any reason for changing the "powered by" here?

Comment (songnguxyz) [reply LocalSettings.php:1595]: I wanted to give the MH icon the personality of my wiki while still giving the hosting platform the credit it deserved. There should be no infringement at all as it still fully shows that MH have a large part of making my wiki possible.

**Review by Reception123 — APPROVED** [review]

**Pull request merged**